danndav/TransportFare
# 167310116: add readme

#### What does this PR do?
This PR add readme file

#### Description of Task to be completed?

  -add readme file

####  How should this be manually tested?
git fetch ch-167310116-add-readme and
checkout to the branch
run the application
navigate to the root folder
click on it to check changes

####  Any background context you want to provide?
N/A

####  What are the relevant pivotal tracker stories?
#167310

####  screenshots (if appropriate)
N/A

####  Questions:
N/A\
